src/Components/DynamicFields.res

```diff
@@ -426,13 +426,22 @@ let make = (
               />
             </div>
           | Currency(currencyArr) =>
+            let updatedCurrencyArray: array<
+              DropdownField.optionType,
+            > = currencyArr->Array.map(item => {
+              let optionVal: DropdownField.optionType = {
```

Comment:
Please create a common function for this

Reply:
Updated